### PR TITLE
Feature/search openpos 36

### PR DESCRIPTION
**Pull request by rexxieboy**
it works

### CODE REVIEW

**Review by jortdr — CHANGES_REQUESTED** [review]
Search will wipe labels (amount ordered) once search value changed. pls fix

**Review by TinyTani — APPROVED** [review]
ja

ja

ideaal

**Review by TinyTani — APPROVED** [review]
ja

ja

ja nee dit is echt top

**Review by VarioCode — APPROVED** [review]

File: OpenPOS-APP/ProductView.xaml.cs [hunk 1]
```diff
@@ -25,12 +26,17 @@ public void SetProductValues(MenuPage page, Product product)
...
       _product = product;
-            
       ProductName.Text = product.Name;
       ProductInfo.Text = product.Description;
       string value = String.Format(((Math.Round(product.Price) == product.Price) ? "{0:0}" : "{0:0.00}"), product.Price);
       ProductPrice.Text = $"€ {value}";
       // ProductImage.Source = imagePath; --Needs to be implemented in DB
+      if (_menuPage.SelectedProducts.ContainsKey(_product.Id))
+        {
+            Debug.WriteLine("asdasd");
```
Comment (jortdr) [inline OpenPOS-APP/ProductView.xaml.cs:36]: :((((((((((( -1

**Review by jortdr — CHANGES_REQUESTED** [review]
Not good.

**Review by jortdr — DISMISSED** [review]
excellent

**Review by jortdr — APPROVED** [review]
ok

**Review by TinyTani — APPROVED** [review]
ja mann

**Pull request merged**